Jenkinsfile

```diff
@@ -0,0 +1,75 @@
+pipeline {
+    agent none
+
+    options {
+          timeout(time: 20, unit: 'MINUTES')
+    }
+    stages {
+        stage('Go build') {
+            agent {
+                docker {
+                    image docker.build("storj-ci", "--pull https://github.com/storj/ci.git").id
+                    args '--user root:root --cap-add SYS_PTRACE -v "/tmp/gomod":/go/pkg/mod '
+                }
+            }
+            steps {
+                script {
+                    // need to keep deleting this directory because of some cleanup issue
+                    sh 'rm -rf tmp-c'
```

Comment:
We should move this into the cleanup step that is always executed.

Reply:
`cleanWs()` already does this.

I am not sure why but there are somtimes permission issues during cleanup.

`java.nio.file.FileSystemException: /var/lib/jenkins/workspace/uplink-php_PR-6/tmp-c/error.go: Operation not permitted`

We can move this line but I fear the issue will return.